### DISCUSSION TITLE
Email test

### DISCUSSION THREAD

**Discussion by mrsparkle331** [outdoorbits/little-backup-box#175] [Q&A] [answered]
Is there any way to send a test email to check the email settings? I'm having issues getting email notifications out. I think a copy triggers an email but I don't see a way to force a test email.

**Comment by outdoorbits** [comment]
Good idea, I will implement this soon and tell you here.

**Comment by outdoorbits** [comment]
Now it is there. After the mail-configuration you can check a checkbox 
to send a testmail. Could you please check that?

**Reply by mrsparkle331** [reply]
I see the new checkbox but I'm not receiving any test emails. 
I don't even see a connection attempt on the mail server side. Is there a place I can see the logs for the email sending?

**Reply by mrsparkle331** [reply]
Also I don't see a place to specify a sender, only a recipient. In my configuration the authentication user to the mail server and the sending address are different.

**Comment by outdoorbits** [comment]
As sender it uses the username, but I will add an option to define a 
sender-address too. But this may take some days. Also it should be not a 
big deal to add mail-logging. I'll tell you here...

**Comment by outdoorbits** [comment, marked as answer]
Now you can configure a sender and get more logging-information about mail transfer if log level is set to three.
